src/test/java/org/apache/commons/collections4/ListUtilsTest.java

```diff
@@ -81,16 +81,25 @@ public void testEmptyIfNull() {
     public void testEquals() {
         final Collection<String> data = Arrays.asList("a", "b", "c");
 
-        final List<String> a = new ArrayList<>( data );
-        final List<String> b = new ArrayList<>( data );
-
-        assertEquals(a, b);
-        assertTrue(ListUtils.isEqualList(a, b));
-        a.clear();
-        assertFalse(ListUtils.isEqualList(a, b));
-        assertFalse(ListUtils.isEqualList(a, null));
-        assertFalse(ListUtils.isEqualList(null, b));
+        final List<String> list1 = new ArrayList<>( data );
```

Comment:
Please don't rename variables. I can't tell what's changed.

Reply:
I renamed them because they overlapped the String constants from lines 46, 47. And they were confusing.
```
private static final String a = "a";
private static final String b = "b";
```
```
final List<String> a = new ArrayList<>( data );
final List<String> b = new ArrayList<>( data );
```